code/modules/admin/verbs/bluespacetech.dm

```diff
@@ -329,6 +329,14 @@
 		C.holder.original_mob = null
 	suicide()
 
+/mob/living/carbon/human/bst/verb/tgm()
```

Comment:
its a convenience. Less hassle than spawning a normal mob. ive done it a lot. 

This just saves a couple seconds of time, which can add up to lots over thousands of testing runs.